### PR TITLE
First pass at new easy-form

### PR DESCRIPTION
**Pull request by bcardarella**
* form-for
* input control
* textarea control

Controls have label and helpers

### CODE REVIEW

**Comment by Trevoke** [comment]
Any idea when this will be merged in / released ?

**Comment by bcardarella** [comment]
@Trevoke it is blocked by some things I need / want to get into ember-cli's build system. Specifically, I want ember-easy-form to be able to "discover" any form-controls that are available and then dynamically create the component template hash as a pre-compile step. In order to do this the way ember-cli is building its assets need to change. I have an early version of the RFC for this here: https://github.com/ember-cli/rfcs/pull/82/files

There are other major benefits to ember-cli exposing the pre-compiled asset graph as a single tree rather than its current piece-meal approach. So I'm hoping to win over some hearts and minds to get this on the roadmap.

tldr; not anytime soon :(

**Comment by k-fish** [comment]
> There are some philosophical differences that motivated me to resurrect easy-form over contributing to form-for

I've been looking at the existing form solutions, and I'm really curious as to what these differences are or what you feel form-for is lacking. Are there parts of it that don't align with current Ember paradigms?

**Comment by bcardarella** [comment]
@k-fish this isn't remotely ready. I am blocked on some upstream changes in ember-cli. If you are looking for something today you should go with form-for

**Comment by k-fish** [comment]
Ah no, I wasn't under the impression that it was ready, I'm just interested in your opinion on form-for 😄  and what the philosophical differences are, considering I was thinking of using it.
